README.md

```diff
@@ -1,56 +1,103 @@
+- `start_transaction/2`
+- `read_objects/3`
+- `update_objects/3`
+- `commit_transaction/2`
+- `abort_transaction/2`
 
-  update a set of object with the specified operations. operation is any allowed (upstream) operation on the crdt type for bound\_object, op\_parameters are parameters to the operation.
-  Example:
+Here is a detailed interface explanation:
 
-           Obj = antidotec_counter:increment(1, antidotec_counter:new()),
-           ok = antidotec_pb:update_objects(Pid,
-                                        antidotec_counter:to_ops(BObj, Obj),
-                                        TxId).
+#### start_transaction(Pid, Timestamp) -> {ok, TxId} | {error, Reason}.
 
-* abort\_transaction (Pid, transaction_descriptor)
-  aborts a transaction
+This function starts a new transaction and returns a transaction identifier, to be used with further operations in the same transaction. Antidote transactions can be of 2 types: _interactive_ or _static_.
 
-* commit\_transaction (Pid, transaction_descriptor) --> {ok,timestamp} OR aborted
+**Static transactions** can be seen as one-shot transactions for executing a set of updates or a read operation. Static transactions do not have to be committed or closed and are mainly handled by the Antidote server.
 
+**Interactive transactions** can combine multiple read and update operations into an atomic transaction. Updates issued in the context of an interactive transaction are visible to read operations issued in the same context after the updates. Interactive transactions have to be committed in order to make the updates visible to subsequent transactions.
 
-## Example
+The default transaction type is `interactive`. Currently you can create `static` transactions by using the alternative `start_transaction/3` callback, that accepts an additional list of transaction properties (you can pass in `[static]` to the third parameter in order to get static transactions).
 
-    %% Starts pb socket
-    {ok, Pid} = antidotec_pb_socket:start(?ADDRESS, ?PORT),
+Here is a simple example:
 
-    BObj = {Key, riak_dt_pncounter, Bucket},
-    Obj = antidotec_counter:increment(Amount, antidotec_counter:new()),
-    {ok, TxId} = antidotec_pb:start_transaction(Pid, term_to_binary(ignore), {}),
-    ok = antidotec_pb:update_objects(Pid,
-                                     antidotec_counter:to_ops(BObj, Obj),
-                                     TxId),
-    {ok, TimeStamp} = antidotec_pb:commit_transaction(Pid, TxId),
-    %% Use TimeStamp for subsequent transactions if required
-    {ok, TxId2} = antidotec_pb:start_transaction(Pid, TimeStamp, [{static=true}]),
-    ...
-    ...
+```erl-sh
+Pid = antidotec_pb_socket:start_link("127.0.0.1", 8087).
+%% ignore leaves out the clock parameter in the following calls
+{ok, TxId1} = antidotec_pb:start_transaction(Pid, ignore). %% interactive transaction
+{ok, TxId2} = antidotec_pb:start_transaction(Pid, ignore, [static]). %% static transaction
```

Comment:
This looks odd. @deepthidevaki Is this true?

Reply:
Got it. I'm going to change it to `{static, true}`.